### ISSUE TITLE
using multiple projects

### ISSUE THREAD

**Issue by MartijnBroens** [evrenvarol/outlook-taskboard#21]
Hi I just downloaded the pckg looks great,

i work on multiple projects at the same time and i was wondering if it would be possibel to have a tag or something in the task identifying the project. if so than perhaps it would also be possible to sort the kaban view per project or for that matter any other tag??

kind regards Martijn

**Comment by BillyMcSkintos** [comment]
Yes, use categories in outlook. Assign each task a category. You can use the search feature to filter. You can also adjust the config to sort by categories
